### PR TITLE
Adding tool to convert resx to csv.

### PR DESCRIPTION
**Pull request by ivannaranjo**
Simple script that extracts the strings from the .resx file and produces a .csv file that can be imported into spreadsheets for easy review of the strings.

### CODE REVIEW

File: tools/resx2csv.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        self.name = root.attrib['name']
+        self.value = root.find('value').text
+        self.comment = root.find('comment').text
+
+
+def convert_to_csv(root, output):
+    """Takes the root of the Xml file and converts it into a .csv file."""
+
+    with open(output, 'wt') as out:
+        out.write("name,value,comment\n")
```
Comment (chrsmith) [inline tools/resx2csv.py]: All other string literals in the file use `'`, whereas this is using `"`.

File: tools/resx2csv.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python
+"""Script to convert the .resx file into a .csv file.
+
+This script will take any .resx file and extract all of the strings to a .csv file
+easily importable into spread sheets.
+
+"""
```
Comment (chrsmith) [inline tools/resx2csv.py]: An example command line for how to invoke the tool would be very useful. e.g.:

python .\tools\resx2csv.py --resx .\src\strings\UiStrings.resx --output ui-strings.txt

Comment (ivannaranjo) [inline tools/resx2csv.py]: Done.

File: tools/resx2csv.py (hunk 1 above)
Comment (ivannaranjo) [inline tools/resx2csv.py]: Cleaned up the literals, old habits die hard.

**Comment by ivannaranjo** [comment]
I am using Python at the moment out of convenience. But I will definitely think about writing these, and future scripts, in Powershell. Now it is more portable than ever before.

**Pull request merged**